tools/java/cpp-build/pom.xml

```diff
@@ -44,16 +64,42 @@
             </goals>
             <configuration>
               <sources>
+                <!-- Add generated/ to the source directories. -->
+                <!-- TODO: Add needed functionality to java/libphonenumber/src/Phonemetadata.java
```

Comment:
Actually I made a mistake about the Maven version - I had thought that it was getting protoc from the protobuf-java dependency but it's not (that's used for the runtime dependencies, not to generate the files).

Changed the comments, I'll push the changes momentarily.